python/social_bot/models/youbot_noplugin/model.sdf

```diff
@@ -537,7 +537,7 @@
       <kinematic>0</kinematic>
     </link>
     <link name="gripper_finger_link_l">
-      <pose>0.213260 -0.001105 0.408579 0.085052 -0.365926 2.733150</pose>
+      <pose>0.218260 0.0099 0.408579 0.085052 -0.365926 2.733150</pose>
```

Comment:
makes distance of left and right gripper a little bit larger. The default distance is too small to grip objects.

Reply:
Please add this as a comment into the model file.